integration_tests/samples/openid_connect/flask_example.py

```diff
@@ -0,0 +1,118 @@
+import json
+import logging
+import os
+import jwt
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+client_id = os.environ["SLACK_CLIENT_ID"]
+client_secret = os.environ["SLACK_CLIENT_SECRET"]
+redirect_uri = os.environ["SLACK_REDIRECT_URI"]
+scopes = ["openid", "email", "profile"]
+
+from slack_sdk.web import WebClient
+from slack_sdk.oauth import OpenIDConnectAuthorizeUrlGenerator, RedirectUriPageRenderer
+from slack_sdk.oauth.state_store import FileOAuthStateStore
+
+state_store = FileOAuthStateStore(expiration_seconds=300)
+
+authorization_url_generator = OpenIDConnectAuthorizeUrlGenerator(
+    client_id=client_id,
+    scopes=scopes,
+    redirect_uri=redirect_uri,
```

Comment:
Unlike the existing authorize URL, redirect_uri is mandatory for building OpenID Connect authorize URL.